### ISSUE TITLE
akıllı bakınız görünümü hakkında

### ISSUE THREAD

**Issue by mehmetkucuk** [thehivecluster/instela-issues#195]
instela'da verilen akıllı bakınızların, itü sözlük'teki akıllı bakınızlar gibi görünmesi planlanıyor mu?

sözlükteki kullanımı hem görsel olarak hem de okumaya hız kazandırması açısından güzel oluyordu. 

http://i.hizliresim.com/pmn8lN.png

**Comment by az3** [comment]
Alakasız bir soru: Sizce akıllı bakınız çok mu gerekli?
Niye var? Evvela mobil ortamlarda okuma deneyimini öldürüyor, iğrenç. 
Gerçekten doğru şekilde kullanan[*](https://www.itusozluk.com/goster.php/kullanmak) çok az insan var. 
Genelde düşünce balonu gibi kullanılıyor, ve yazıda konseptin dışına çıkıp alakasız fikirleri serpiştirmekte[*](https://www.itusozluk.com/goster.php/asdasdasd+bunu+okudun)[*](https://www.itusozluk.com/goster.php/bunuda)[*](https://www.itusozluk.com/goster.php/da+ayr%FD) kullanılıyor.
O özelliğin tamamen kaldırılması düşünülebilir.
Geriye dönük olarak, mevcut kullanılmış akıllı bakınızlar da, "(bkz: swh)" şeklinde normal bakınıza dönüştürülebilir.

**Comment by sageeth** [comment]
Kesinlikle katılıyorum. Düşünce balonu ve "swh" kullanımından başka kullanılmaz oldu. Tamamen kaldırılması fikrini olumlu buluyorum.

**Comment by collonsdedeu** [comment]
geriye dönük akıllı bakınızlar da (bkz: swh) formatına dönüşecekse onun da masaüstü okuma deneyimini öldüreceğini düşünüyorum. zira metnin içindeki bakınızlar bile yeterince sırıtıyor, metnin akışına zarar vermeden, formata uygun kullanılan akıllı bakınızların yaşayacağı dönüşüm de akıcılığı engelleyecektir. mesela bir örnek: https://www.itusozluk.com/goster.php/@3036096 bu metindeki akıllı bakınızlar normal bakınıza dönüşürse okumak zulüm hale gelecektir. en başında akıllı bakınız olmasa ve herkes buna göre metni ele almış olsa neyse de, geriye dönük metinleri değiştirmek okunabilirlik açısından sıkıntı. ekşi'nin mobil sürümünde akıllı bakınızlar (*:swh) formatında görüntüleniyor, masaüstünde normal bir şekilde. sadece mobile yönelik bir değişim daha akılcı duruyor bana kalırsa.

bir de yanlış anlamayın ama millet kullanmayı bilmiyor diye akıllı bakınızı kaldırmak yaralı parmağı kesmekten farksız. akıllı bakınızın hıyarca kullanımı diye bir ispiyon maddesi var, bunun içi doldurulursa sıkıntı olacağını düşünmüyorum.

**Comment by sageeth** [comment]
akıllı bakınızın hıyarca kullanımı ispiyonları için değerlendirmenin netleştirilmesi lazım. çok öncelerden bu sebeple ispiyon yapardım ancak hep reddedilirdi, artık o tarz girileri ispiyonlamıyorum bile. net bir tavır yok sanırım bu konuda.

"... okuduğunu anlar!:okuduğunu anlamak!: demiştim..." şeklindeki kullanımlar dışındaki tüm düşünce balonları ve bana göre amaçsız bir kullanım olan "swh" kullanımları silinme nedeni olmalı. akıllı bakınızda swh/gab/ahaha gibi kullanımların düşünce balonu içeriğine alınması ve silinmesi gerekir bence.

**Comment by collonsdedeu** [comment]
aynen, ispiyon değerlendirmelerinde bir standart, açıkça belirtilmiş kriterler yok. varsa bile çoğu yoruma açık olduğu için uygulama alanında sıkıntı yaşanıyor. akıllı bakınız ya da başka bir konuda hatalı girilerin silinmesinden endişe edilmemesi lazım. hatalı girisi silinmiş bir yazar "ben nerede yanlış yaptım" şeklinde kendini sorgulamak yerine sözlükte "lanet federaller girimi silmişler" şeklinde ağlaşıyor diye moderasyon faşist olmuyor. bu tip, özeleştiri yapmayan, sadece girisinin silinip silinmediğiyle, artı oy alıp almadığıyla ilgilenen kullanıcıların sözlükte daha çok tüketici rolünde olduklarını düşünüyorum. tüketici yazarların ayağını kesmemek adına birtakım serbestlikler tanıyıp üreticileri de sözlükten kaçırmamak lazım diye düşünüyorum. akıllı bakınızın varlığı ve doğru kullanımı tek başına sözlüğü okunası bir yer haline getirmiyor ama bu tip ufak şeyler birleşerek okunası bir yapı oluşturuyor.

konu dağıldı, bari issue'yu açan arkadaşa şunu sunayım: #167 birkaç gün önce akıllı bakınız içeriği görüntülenmiyordu, şu anki zaten geliştirilmiş hali. görselliği zamanla değişebilir, şu anda kullanımında bir sıkıntı yok.

**Comment by mehmetkucuk** [comment]
şimdi bazı sözlük özellikleri 10 yıldır kullanılageldiği için bir sözlük yazarı olarak öncelikle sorduğum sorunun gerekli ve cevaba değer olduğunu düşünüyorum. ekşi sözlük'ün yeni tasarıma geçerken frame yapılı site için "eski dandik site" demesi gibi muamele etmezsek eğer itü sözlük'e bence iyi olacaktır. 

o zaman "instela nedir? nasıl kullanılır? sözlükten bağımsız, belli bir formata dayanmayan bir sosyal site midir?" bunları açıklamak gerekebilir insanlara. 

içinde "swh", "gab", "gb", "gülen sımayli" yazan bakınızları tek tek "(bkz: swh)" ye çevirmek yerine akıllı bakınızları içerikleriyle beraber silmek daha yerinde bile olur. peki bu durumda akıllı bakınızı gülmek için ya da hıyarca kullanmayan binlerce yazar mağdur olmaz mı?

**Comment by sayz** [comment]
ya hu niye bu kadar kaldırma meraklısıyız onu merak ediyorum? örneğin ekşi sözlükte akıllı bakınızlar bilgisayarda [*](https://eksisozluk.com/?q=swh) şeklinde görünütülenirken aynı entry'i mobilden görüntelidiğinizde [(*: swh)](https://eksisozluk.com/?q=swh) şeklinde görüntüleniyor. 

açıkçası bi görüntü kirliliği falan göremiyorum ben. hem şu hıyarca kullanım şeysini hiç anlamıyorum ya hu bırakın insanlar kullansın o şekilde ne olacak? yani bi zorlama yok bi şey yok burada sonuçta kimseye.
